pallets/creditcoin/src/mock.rs

```diff
@@ -328,23 +345,22 @@ pub fn roll_to_with_ocw(n: BlockNumber) {
 	let now = System::block_number();
 	for i in now + 1..=n {
 		System::set_block_number(i);
-		Creditcoin::on_initialize(i);
-		Creditcoin::offchain_worker(i);
-		Creditcoin::on_finalize(i);
+		TaskScheduler::on_initialize(i);
+		TaskScheduler::offchain_worker(i);
+		TaskScheduler::on_finalize(i);
 	}
 }
 
 #[allow(dead_code)]
 pub fn roll_by_with_ocw(n: BlockNumber) {
 	let mut now = System::block_number();
 	for _ in 0..n {
-		Creditcoin::offchain_worker(now);
+		TaskScheduler::offchain_worker(now);
 		now += 1;
 		System::set_block_number(now);
 		System::reset_events();
 		System::on_initialize(now);
-		Creditcoin::on_initialize(now);
-		Creditcoin::on_finalize(now);
+		TaskScheduler::offchain_worker(now);
```

Comment:
Won't this call the offchain worker hook twice per block? (Once on this line, and then again on the start of the next loop iteration)